apps/storybook-react/stories/WalletHome.stories.tsx

```diff
@@ -55,7 +55,7 @@ const WalletHome: React.FC = () => {
       {/* Container Expanded View */}
       <Box className="mx-auto w-full bg-default md:max-w-xl md:rounded-3xl md:py-4">
         {/* Header */}
-        <Box className="border-color-border-muted px-4 py-4 md:px-8">
+        <Box className="border-b border-muted p-4 md:px-8">
```

Comment:
`no-custom-classname` this rule is one of the most important IMO it catches invalid tailwind classnames. `border-color-border-muted` was invalid but it was not being picked up.